saltswap/swapper.py

```diff
@@ -661,13 +651,9 @@ def attempt_identity_swap(self, context, penalty, saltmax=None):
         if mode == "remove salt":
             self.proposal = [ncation, ncation - 1]
             self.cumulative_work = cumulative_work
```

Comment:
Do you no longer want to track differences between work spent removing salt, and work spent adding salt?